components/automate-ui/src/app/pages/api-token/details/api-token-details.component.scss

```diff
@@ -40,6 +40,11 @@ chef-tab-selector {
 
 chef-form-field {
   width: 240px;
+  padding-bottom: 30px;
+}
+
+chef-radio chef-option {
+  display: inline-block;
```

Comment:
Only thing to override: convert from vertical to horizontal
(Text color upon select will be addressed by fixing the control, in a separate task.)